src/main/kotlin/com/vk/admstorm/actions/SendLogsToHastebinAction.kt

```diff
@@ -12,13 +12,19 @@ class SendLogsToHastebinAction : AdmActionBase() {
         val project = e.project ?: return
         val hasteLink = HastebinService.getInstance(project).createHaste(readIdeaLogFile())
 
-        AdmNotification("Thanks for logs!")
-            .withTitle("Logs successfully sent to Hastebin")
-            .withActions(AdmNotification.Action("Copy hastebin link") { _, notification ->
-                MyUtils.copyToClipboard(hasteLink)
-                notification.expire()
-            })
-            .show(project)
+        val notification = if (hasteLink != null) {
```

Comment:
bring back the old code. And before you add:
```kotlin
if (hasteLink == null) {
    AdmNotification()
        .withTitle("Hastebin service unavailable. Try again later")
        .show(project)
    return
}
```

Reply:
Done